services/navigation/navigation.go

```diff
@@ -64,23 +65,32 @@ func FromRobot(r robot.Robot, name string) (Service, error) {
 
 // Config describes how to configure the service.
 type Config struct {
-	Store              StoreConfig `json:"store"`
-	BaseName           string      `json:"base"`
-	MovementSensorName string      `json:"movement_sensor"`
-	DegPerSecDefault   float64     `json:"degs_per_sec"`
-	MMPerSecDefault    float64     `json:"mm_per_sec"`
+	Store               StoreConfig                   `json:"store"`
+	BaseName            string                        `json:"base_name"`
+	MovementSensorName  string                        `json:"movement_sensor_name"`
+	MotionServiceName   string                        `json:"motion_service_name"`
+	DegPerSecDefault    float64                       `json:"degs_per_sec"`
+	MetersPerSecDefault float64                       `json:"meters_per_sec"`
+	Obstacles           spatialmath.GeoObstacleConfig `json:"obstacles,omitempty"`
```

Comment:
my reasoning is because its the only struct that can handle the specification of GeoObstacles. 
Is there a way to do this without defining another struct? 👀

Reply:
But how is this struct even being called/used in the first place?  We shouldn't have two configs with the same information.  Either we should remove this and use the one in builtin, or within builtin we should be referencing the one defined in this package.